orderbook/src/main.rs

```diff
@@ -159,8 +160,11 @@ async fn main() {
         None
     };
 
-    let event_updater =
-        EventUpdater::new(settlement_contract.clone(), database.clone(), sync_start);
+    let event_updater = EventUpdater::new(
+        settlement_contract.clone(),
+        database.as_ref().clone(),
```

Comment:
Did something change on the updater input parameters or is the database struct just different now?